modules/rosa-understanding-deployment-modes.adoc

```diff
@@ -6,7 +6,7 @@
 [id="rosa-understanding-deployment-modes_{context}"]
 = Understanding the auto and manual deployment modes
 
-When installing a {product-title} (ROSA) cluster that uses the AWS Security Token Service (STS), you can choose between the `auto` and `manual` ROSA CLI (`rosa`) modes to create the required AWS Identity and Access Management (IAM) resources.
+When installing a {product-title} (ROSA) cluster that uses the AWS Security Token Service (STS), you can choose between the `auto` and the `manual`  {product-title} (ROSA) CLI (`rosa`) modes to create the required AWS Identity and Access Management (IAM) resources.
```

Comment:
Honestly this can probably just say "you can choose between the `auto` and `manual` modes to create..." rather than mentioning the CLI specifically at all